packages/node/src/lib/integrations/integrations.interface.ts

```diff
@@ -1,5 +1,17 @@
-import { ChannelTypeEnum, ICredentialsDto } from '@novu/shared';
+import {
+  BuilderFieldType,
+  BuilderGroupValues,
+  ChannelTypeEnum,
+  FilterParts,
+  ICredentialsDto,
+} from '@novu/shared';
 
+interface ICondition {
```

Comment:
```suggestion
export interface IConditions {
```
add an s - To be consistent with the name in the fe 
+ export the type